### PR TITLE
MW-224 -10/16 Changes

### PR DESCRIPTION
**Pull request by bcanedo4**
Dot Carousel and Automatic Carousel are finished.

### CODE REVIEW

File: src/sass/_carousel.scss [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  position: absolute;
+  width: 100%;
+  text-align: center;
+}
+
+.darken {
+  filter: brightness(50%);
+}
+
+/* Page specific */
```
Comment (jasmine1569) [inline src/sass/_carousel.scss]: These are not notes, when reviewing them in the code they are not fade out. Also need to be more detail in your notes.

File: src/sass/_carousel.scss [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+}
+
+.auto-carousel--container {
+  max-width: 1000px;
+  position: relative;
+  margin: auto;
+  text-align: center;
+}
+
+/* Success Story Page */
```
Comment (jasmine1569) [inline src/sass/_carousel.scss]: These are not notes, when reviewing them in the code they are not fade out. Also need to be more detail in your notes.

File: src/sass/_carousel.scss [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+  to {opacity: 1}
+}
+
+@keyframes fade {
+  from {opacity: .4}
+  to {opacity: 1}
+}
+
+
+/* Automatic Carousel */
```
Comment (jasmine1569) [inline src/sass/_carousel.scss]: These are not notes, when reviewing them in the code they are not fade out. Also need to be more detail in your notes.

File: src/sass/_carousel.scss [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+.active {
+  background-color: #060707;
+}
+
+.dot:hover {
+  background-color: #fef8ff;
+}
+
+/* Fading animation */
```
Comment (jasmine1569) [inline src/sass/_carousel.scss]: These are not notes, when reviewing them in the code they are not fade out. Also need to be more detail in your notes.

File: src/sass/_carousel.scss [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+}
+
+.carousel--container {
+  max-width: 50%;
+  position: relative;
+  margin: auto;
+  text-align: center;
+}
+
+/* The dots/bullets/indicators */
```
Comment (jasmine1569) [inline src/sass/_carousel.scss]: These are not notes, when reviewing them in the code they are not fade out. Also need to be more detail in your notes.

File: src/sass/_carousel.scss [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+/* Fading animation */
+.fade {
+  -webkit-animation-name: fade;
+  -webkit-animation-duration: 1.5s;
+  animation-name: fade;
+  animation-duration: 1.5s;
+}
+
+@-webkit-keyframes fade {
+  from {opacity: .4}
```
Comment (jasmine1569) [inline src/sass/_carousel.scss]: what is from? the code don't like it

File: src/sass/_carousel.scss [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+  to {opacity: 1}
```
Comment (jasmine1569) [inline src/sass/_carousel.scss]: what is to? the code don't like it

**Comment by RahilHashim** [comment]
merge dev into your branch and fix merge conflicts as well

**Comment by RahilHashim** [comment]
clean up the code. Why so many containers nested into containers? follow you convention that you put in the grid system.

**Pull request merged**